client/src/env.ts

```diff
@@ -8,21 +8,25 @@ export const DISABLE_AUTH = Boolean(
   JSON.parse(process.env.REACT_APP_DISABLE_AUTH || "false")
 );
 
-export const CRUD_MODE = Boolean(
-  JSON.parse(
-    process.env.REACT_APP_CRUD_MODE ||
-      JSON.stringify(process.env.NODE_ENV === "development")
-  )
+export const WRITER_MODE = Boolean(
+  JSON.parse(process.env.REACT_APP_WRITER_MODE || "false")
 );
 
-export const CRUD_MODE_HOSTNAMES = (
-  process.env.REACT_APP_CRUD_MODE_HOSTNAMES ||
+export const WRITER_MODE_HOSTNAMES = (
+  process.env.REACT_APP_WRITER_MODE_HOSTNAMES ||
   "localhost,localhost.org,127.0.0.1"
 )
   .split(",")
   .map((x) => x.trim())
   .filter(Boolean);
 
+export const DEV_MODE = Boolean(
+  JSON.parse(
+    process.env.REACT_APP_DEV_MODE ||
+      JSON.stringify(process.env.NODE_ENV === "development")
```

Comment:
I wonder if `NODE_ENV` is actually `development` in the context of mdn/content. 🤔

Reply:
So likewise, maybe we could default to `!import.meta.url.contains('@mdn/yari')`?

Reply:
Honestly I think we can just remove this `NODE_ENV` check, now that I've added `REACT_APP_DEV_MODE` to `.env-dist` - all it seemed to be doing was turning `CRUD_MODE` on by default in `start:client` in `package.json`